.github/workflows/build-and-lint-front.yml

```diff
@@ -16,5 +16,7 @@ jobs:
           node-version: 18.15.0
           cache: "npm"
           cache-dependency-path: ./front/package-lock.json
+      - working-directory: types
+        run: npm install && npm run lint &&  npm run build
```

Comment:
can we skip some of it to keep front fast? I presume lint for sure? Maybe even build?